docs/ai/toc.yml

```diff
@@ -81,12 +81,14 @@ items:
   items:
   - name: The Microsoft.Extensions.AI.Evaluation libraries
     href: conceptual/evaluation-libraries.md
-  - name: "Quickstart: Evaluate the quality of a response"
-    href: quickstarts/evaluate-ai-response.md
-  - name: "Tutorial: Evaluate the safety of a response"
-    href: tutorials/evaluate-safety.md
-  - name: "Tutorial: Evaluate a response with caching and reporting"
-    href: tutorials/evaluate-with-reporting.md
+  - name: Tutorials
+    items:
+    - name: "Quickstart: Evaluate the quality of a response"
+      href: quickstarts/evaluate-ai-response.md
+    - name: "Evaluate response quality with caching and reporting"
+      href: tutorials/evaluate-with-reporting.md
+    - name: "Evaluate response safety with caching and reporting"
```

Comment:
Would this also change the titles displayed at the top of the pages? If not, should we fix those too?